### ISSUE TITLE
Support for heart rate targets?

### ISSUE THREAD

**Issue by joschkazj** [ManuelWiese/mrc_tools#3]
Hi,
this tool looks very interesting as I'm planning to create workouts locally or using Trainerday's editor and saving them to my Elemnt Bolt via USB.
Since you've done some investigating I'd like to know if these file formats support setting heart rate targets for the Bolt or what is missing to use these.

Thanks in advance for your support
Joschka

**Comment by n-bock** [comment]
Yes, you can set `HR_LO=160` and `HR_HI=170` for example to pass the heart rate range 160 to 170 bpm to a specific interval.
